test/cli/samples/warn-import-export/_config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
 		assertIncludes(
 			stderr,
 			'(!) Unused external imports\n' +
-				"default imported from external module 'external' but never used\n"
+				`default imported from external module 'external' but never used in "main.js"\n`
```

Comment:
While you are at it, maybe we could align on using double quotes `"` also for the module name. This is also what most other errors use.